src/routes/[id]/training/+layout.server.ts

```diff
@@ -23,7 +23,28 @@ export const load: LayoutServerLoad = async ({ parent }) => {
     memberOfQueue = null;
   }
 
+  let position = -1;
```

Comment:
I know this isn't code you wrote, but since I'm reviewing it, I'm going to complain about it: the way we get the queue can be optimized to a single query using a join.
Furthermore, instead of making memberOfQueue null and doing all of that, we should just early return with empty data if the query returns nothing.

Reply:
36bdb0869c7746d3ec578c7f3a8795b05d24e40e

Still returns null in this one, but gets rid of the unneeded query